### ISSUE TITLE
Chinese input method automatically disappear, but the text did not enter.

### ISSUE THREAD

**Issue by zjfjiayou** [ianstormtaylor/slate#2349]
#### Do you want to request a _feature_ or report a _bug_?
Bug

#### What's the current behavior?
Environment： 
-   Windows 7 64
-   Chrome 49
-   slate  0.43.0
-   slate-react 0.20.1

The following happens when I type Chinese in the last line of the document

![qq20181030-143106](https://user-images.githubusercontent.com/7593789/47701087-a7c67e80-dc53-11e8-9b9a-87fdfdab292d.gif)

Chinese input method automatically disappear, but the text did not enter. I turned on debug mode and found that `updateSelection` was triggered after triggering `onCompositionStart`. So I'm blocking `this.UpdateSelection() `from the following code,However, I do not know what other problems will arise when this code is blocked, so I would like to ask you to help solve this problem or provide a compatible solution

```javascript
      if (range && range.equals(selection.toRange())) {
        this.updateSelection()
        return
      }
```

#### What's the expected behavior?
Input Chinese correctly at the end

**Comment by zjfjiayou** [comment]
I created an example here https://github.com/zjfjiayou/slate-test/tree/master
You can activate him with the following command
npm i && npm start 
Before that, you'll need to prepare for Windows 7 and chrome 49, which you can download here
https://www.slimjet.com/chrome/download-chrome.php?file=win%2Fchrome64_49.0.2623.75.exe

**Comment by ianstormtaylor** [comment]
Closing in favor of https://github.com/ianstormtaylor/slate/issues/2368, if you think this is unrelated let me know and we can reconsider.

**Comment by zhengjunxin** [comment]
@zjfjiayou have you fixed this bug?

**Comment by giddens9527** [comment]
Also following this issue, saw one possible fix here: #2415

**Comment by zhengjunxin** [comment]
@giddens9527 It help me! Thanks
